src/python/WMCore/MicroService/Unified/MSMonitor.py

```diff
@@ -0,0 +1,134 @@
+"""
+File       : MSMonitor.py
+Author     : Valentin Kuznetsov <vkuznet AT gmail dot com>
+             Alan Malta <alan dot malta AT cern dot ch >
+Description: MSMonitor class provide whole logic behind
+the transferor monitoring module.
+"""
+# futures
+from __future__ import division, print_function
+
+# WMCore modules
+from WMCore.MicroService.Unified.MSCore import MSCore
+
+
+class MSMonitor(MSCore):
+    """
+    MSMonitor class provide whole logic behind
+    the transferor monitoring module.
+    """
+    def execute(self, reqStatus):
+        """
+        Executes the MS monitoring logic
+        :param reqStatus: request statue to process
+        :return:
+        """
+        try:
+            # get requests from ReqMgr2 data-service for given statue
+            # here with detail=False we get back list of records
+            requests = self.reqmgr2.getRequestByStatus([reqStatus], detail=False)
+            self.logger.debug('+++ monit found %s requests in %s state',
+                              len(requests), reqStatus)
+
+            # FIXME: completion threshold should come either from unified
+            # or campaign configuration
+            thr = self.unifiedConfig().get('completedThreshold', 100)
```

Comment:
please comment this line and hard-wire it to 100 (I'll come back to it tomorrow or so), otherwise...
```
2019-08-01 16:04:03,496:ERROR:MSMonitor: +++ monit error: 'list' object has no attribute 'get'
Traceback (most recent call last):
  File "/data/srv/HG1908d/sw/slc7_amd64_gcc630/cms/reqmgr2ms/0.1.2.pre3/lib/python2.7/site-packages/WMCore/MicroService/Unified/MSMonitor.py", line 35, in execute
    thr = self.unifiedConfig().get('completedThreshold', 100)
AttributeError: 'list' object has no attribute 'get'
```

Reply:
done